README.md

```diff
@@ -206,6 +206,22 @@ def my_opt_changed():
 
 This layer currently does not define any actions.
 
+# Exec.d Support
+
+This is a mechanism used to customize provisioned machines so the charms can
+actually work. This is useful if there are pre-conditions in your cloud that
+you must account for, such as charm pre-requisits (pre-reactive install) or
+environment customization.
+
+The feature implementation documented this as the following:
```

Comment:
Maybe call this an example use-case instead?  I'm not clear on what "the feature implementation documented this as" means.